packages/composites/gm-token/src/endpoint/price.ts

```diff
@@ -56,4 +55,26 @@ export const endpoint = new AdapterEndpoint({
   name: 'price',
   transport: gmTokenTransport,
   inputParameters,
+  customInputValidation: (req): AdapterInputError | undefined => {
+    const { index, long, short } = req.requestContext.data
+    const indexToken = tokenAddresses.arbitrum[index as keyof typeof tokenAddresses.arbitrum]
+    const longToken = tokenAddresses.arbitrum[long as keyof typeof tokenAddresses.arbitrum]
+    const shortToken = tokenAddresses.arbitrum[short as keyof typeof tokenAddresses.arbitrum]
```

Comment:
```suggestion
    const indexToken = tokenAddresses.arbitrum[index.toUpperCase() as keyof typeof tokenAddresses.arbitrum]
    const longToken = tokenAddresses.arbitrum[long.toUpperCase() as keyof typeof tokenAddresses.arbitrum]
    const shortToken = tokenAddresses.arbitrum[short.toUpperCase() as keyof typeof tokenAddresses.arbitrum]
```

Reply:
Actually noticed that `WBTC.b` isn't uppercase, so this doesn't make sense. Closing!